Makefile

```
@@ -345,7 +345,7 @@ install-files: dbus-install install-conf install-plugins install-post-boot insta
 	elif [ -f /etc/redhat-release ]; then \
 		install etc-conf/rhsmcertd.init.d \
 			$(DESTDIR)/etc/rc.d/init.d/rhsmcertd; \
-	else \
+	elif [ "$(shell cat /etc/SuSE-release | grep VERSION | awk '{ print $$3 }')" == "11" ]; then \
```

Extra $ in the awk piece.
```suggestion
	elif [ "$(shell cat /etc/SuSE-release | grep VERSION | awk '{ print $3 }')" == "11" ]; then \
```